docs/additional-features/napalm.md

```diff
@@ -0,0 +1,65 @@
+# NAPALM
```

Comment:
Thank you for adding this documentation! It's been missing for a long time, yet somehow was never directly addressed.

Reply:
Right there with ya! If there's anything else burning that you'd like written, I'm here to help :). It might not be how you want it from the get go, but I accept constructive criticism with open arms.